### DISCUSSION TITLE
Can you input checkboxes in the description when creating issue?

### DISCUSSION THREAD

**Discussion by KnightNight010** [python-gitlab/python-gitlab#2257] [Q&A] [answered]
## Description of the problem, including code/CLI snippet
Im quite new to this, so please bear with me. Simple question; can I input checkboxes with this module? I have been trying to create issues with a script, which has been going well enough, except it seems not to be able to interpret checkboxes. Can it be done?

Any help is appreciated!

`p.issues.create(
    {'title': 'Test',
     'description': '[ ] test',
     'labels': ['Backlog','Prio 1'],
     'weight': '2'})
`

## Expected Behavior
Input a checkbox with 'test' next to it

## Actual Behavior
Literally inputs '[ ]' 

## Specifications

  - python-gitlab version: 3.8.1
  - API version you are using (v3/v4): v4
  - Gitlab server version (or gitlab.com): unknown

**Comment by JohnVillalovos** [comment, marked as answer]
You will need to figure out how GitLab does checkboxes using markdown.

Maybe this: https://docs.gitlab.com/ee/user/markdown.html#task-lists

And then put the markdown into the `description`

**Reply by KnightNight010** [reply]
I will try that, thank you!

**Reply by KnightNight010** [reply]
I got it working! Thank you again!
